Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rust
+LABEL Name=tinychain Version=0.0.1
+RUN apt-get -y update && apt-get install -y sudo
+
+# Timezone Setting
+ENV TZ=America/New_York
```

Comment:
Is there a way we could make this user-configurable?

Reply:
Yes